src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/ActiveDirectoryTests.ps1

```diff
@@ -695,6 +696,37 @@ function Test-CreateDeleteAppPasswordCredentials
     $credCount = $cred2 | where {$_.KeyId -in $cred1.KeyId, $cred.KeyId}
     Assert-AreEqual $credCount.Count 2
 
+	# Add 1 key credential to the same app
```

Comment:
@grlin You added a lot of tests here, but didn't re-record the test.  This appears to be why the PR is failing